ui/admin/app/components/form/worker/create-worker-led/index.hbs

```diff
@@ -5,42 +5,54 @@
   @showEditToggle={{false}}
   as |form|
 >
-  <form.input
-    @name='cluster_id'
-    @value={{this.clusterId}}
-    @label={{t 'resources.worker.form.cluster_id.label'}}
-    @helperText={{t 'resources.worker.form.cluster_id.help'}}
-    @linkText={{t 'actions.learn-more'}}
-    @link={{doc-url 'worker.manage-workers'}}
-    placeholder='69b6ddb3-ffec-42ab-a994-f43a6519470a'
-  />
+  <h3>{{t 'resources.worker.form.steps.1.title'}}</h3>
+  <div class='worker-create-section'>
```

Comment:
I am wondering what is the purpose of this div and the new introduced class. Are we going to use it to give specific styles?

Reply:
It's just some custom CSS to handle the custom styles for this form since it's a bit different from our `fieldset` component and we had some custom styling for things like icons